frontend/src/constants.ts

```diff
@@ -1,7 +1,7 @@
 export const GO_BACK = -1 as const;
 export const BASE_URL = import.meta.env.VITE_BASE_URL || "http://localhost:5001";
-export const MIN_PRICE = 0 as const;
-export const MAX_PRICE = 10000000 as const;
+export const MIN_PRICE = 0 as number;
```

Comment:
casting in this case is unnecessary, the value `0` already infers type number. All variables defined here should be constants so I would cast it `as const`

Reply:
Done